cpp/include/cuml/cluster/spectral.hpp

```diff
@@ -36,7 +36,8 @@ namespace Spectral {
    * @param out output array for embedding (size n*n_comonents)
    */
 void fit_embedding(const raft::handle_t &handle, int *rows, int *cols,
-                   float *vals, int nnz, int n, int n_components, float *out);
+                   float *vals, int nnz, int n, int n_components, float *out,
```

Comment:
Just wanted to leave a comment that one CI job showed a small doxygen issue, so that it doesn't block things later:

```bash
Generating docs for c/workspace/cpp/include/cuml/cluster/spectral.hpp:38: error: The following parameter of 
ML::Spectral::fit_embedding(const raft::handle_t &handle, int *rows, int *cols, float *vals, int nnz, int n, int n_components, float 
*out, unsigned long long seed=1234567) is not documented:
17:47:37   parameter 'seed' (warning treated as error, aborting now)
```